File: Python/Logic-2/close_far.py
def close_far(a, b, c):
  if abs(a-b) <= 1 and abs(a-c) > 1 and abs(b-c) > 1:
    return True
  elif abs(a-c) <= 1 and abs(a - b) > 1 and abs(b-c) > 1:
    return True
  elif abs(b-c) <= 1 and abs(a-c) > 1 and abs(a-b) > 1:
    return True
  elif abs(a-b) > 1 and abs(a-c) > 1 and abs(b-c) > 1:
    return False
  elif abs(a-b) <= 1 and abs(a-c) <= 1 and abs(b-c) <=1:
    return False
  return False

File: Java/Logic-2/closeFar.java
public boolean closeFar(int a, int b, int c) {
  if(isCloseFar(a,b) && isCloseFar(b,c) && isCloseFar(a,c)) return false;
  if((isCloseFar(a,b) && isCloseFar(b,c)) | (isCloseFar(a,b) && isCloseFar(a,c)) |(isCloseFar(b,c) && isCloseFar(a,c))) return false;
  return isCloseFar(a,b) | isCloseFar(b,c) | isCloseFar(a,c);
}

public boolean isCloseFar(int n, int m){
  if(Math.abs(n-m) >= 2) return false;
  return true;
}

